scanners/pageload.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from phantomas import Phantomas, PhantomasError
+
+command = Phantomas.CMD
+init = None
+
+def scan(domain, options):
+    logging.debug("[%s][pageload]" % domain)
+
+    if not (domain.startswith('http://') or
+            domain.startswith('https://')):
+        domain = 'http://' + domain
```

Comment:
Yeah, this was a bit of a hack because Phantomas needs the full URL (w/ scheme) to work properly, and I wanted to maintain compatibility with the other scanners (e.g. `scan --scanners=tls,pageload whitehouse.gov` should work). If the `inspect` task is already determining the endpoint in a more intelligent way, then I think it'd be great to re-use those results here.

My only concern is that it would create a dependency between two of the scanners, and my thinking was that they should all be independent. However, we could easily use the `inspect` data if it is available (as you do in the linked TLS scanner code), and otherwise fall back onto this existing logic.